src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,16 +1,33 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
     # TO-DO
-    
+    for i in range(elements):
+        if arrA[i] < arrB[i]:
+            elements.append(arrA[merged_arr])
+        elif arrB[i] == merged_arr[i]:
+            arrA[i] += merged_arr[i]
```

Comment:
Should be assigning merged_arr[i] to arrA[i]. I believe you had fixed this in the next commit.